tests/ecosystem/upgrade/test_resources.py

```diff
@@ -90,3 +97,28 @@ def test_pod_io(
         helpers.wait_for_resource_state(pod, constants.STATUS_RUNNING, timeout=600)
         job_name = pod.get_labels().get("job-name")
         job_obj.delete(resource_name=job_name)
+
+
+@post_upgrade
+@bugzilla("1974343")
+@pytest.mark.polarion_id("OCS-2629")
+def test_pod_log_after_upgrade():
+    """
+    Check OSD/MON/MGR pod logs after upgrade and verify the expected log exist
+
+    """
+    pod_objs = get_osd_pods() + get_mon_pods() + get_mgr_pods()
+    pod_names = [osd_pod_obj.name for osd_pod_obj in pod_objs]
+    expected_log_after_upgrade = "set uid:gid to 167:167 (ceph:ceph)"
+    logging.info(
+        f"The log '{expected_log_after_upgrade}' appears after the osd/mon/mg pod is initialized"
```

Comment:
This message is printed before the check is already done, right? Wouldn't be more clear to add something like _Check that the log ..._?

Reply:
done